kata-deploy/README.md

```diff
@@ -0,0 +1,115 @@
+# kata-deploy
+
+[kata-deploy](kata-deploy) provides a Dockerfile which contains all of the binaries
+and artifacts required to run Kata Containers, as well as reference daemonsets which can be utilized to install Kata Containers on a running Kubernetes cluster.
+
+Note, installation via daemonsets will only succesfully install `kata-runtime` on
+a node if it is using either containerd or CRI-O CRI-shims.
+
+## Quick start:
+
+### Installing Kata on a running Kubernetes cluster
+
+```
+kubectl apply -f kata-rbac.yaml
+kubectl apply -f kata-deploy.yaml
+```
+
+### Running a sample workload:
+
+Untrusted workloads can node-select based on ```kata-runtime=true```, and will be
+run via kata-runtime if they are marked with the appropriate CRIO or containerd
+annotation:
+```
+CRIO:           io.kubernetes.cri-o.TrustedSandbox: "false"
+containerd:     io.kubernetes.cri.untrusted-workload: "true"
+```
+
+A sample workload for running untrusted on a kata-enabled node:
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+   annotations:
+    io.kubernetes.cri-o.TrustedSandbox: "false"
+    io.kubernetes.cri.untrusted-workload: "true"
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
+    kata-runtime: "true"
+```    
+
+To run:
+```
+kubectl apply -f examples/nginx-untrusted.yaml
+```
+
+The user should see the pod start, can verify that the pod is indeed making use of
+Kata Runtime by comparing the container ID observed by:
+```
+/opt/kata/bin/kata-runtime list
+kubectl describe pod nginx-untrusted
+```
+
+To remove the test pod:
+```
+kubectl delete -f examples/nginx-untrusted.yaml
+```
+
+### Removing Kata from the Kubernetes cluster:
+
+```
+kubectl delete -f kata-deploy.yaml
+kubectl apply -f kata-cleanup.yaml
+kubectl delete -f kata-cleanup.yaml
+kubectl delete -f kata-rbac.yaml
+```
+
+## kata-deploy Details
+
+### Dockerfile
+
+A Dockerfile is created which contains all of the necessary artifacts for running
+Kata Containers.
+
+Host artifacts:
+* kata-runtime: pulled from Kata github releases page
```

Comment:
Capitalize "github" in all instances: * kata-runtime: pulled from Kata GitHub releases page